components/breadcrumb/stories/breadcrumb.stories.js

```diff
@@ -1,9 +1,18 @@
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
 import { version } from "../package.json";
 import { BreadcrumbGroup } from "./breadcrumb.test.js";
+import { Template } from "./template";
 
 /**
  * Breadcrumbs show hierarchy and navigational context for a user's location within an app.
+ * 
+ * ## Nesting
+ * Breadcrumbs truncate when there is not enough room to display all levels of the breadcrumb list, or as a way of managing relevance of the visible breadcrumb items in a deeply nested hierarchy. The truncation of breadcrumb items begins when either there is not enough room to display all items, or if there are 5 or more breadcrumbs to display.
+ * 
+ * The nested variants below are non-functional. Implementations can add their own overflow menus to display all options within a breadcrumb.
```

Comment:
IMO this is such a huge improvement over anything that the Docs Site or Storybook had before! I also appreciate that you mention that these are not functional and that overflow menus could be added to display all the options.

As an optional suggestion for making these even better, I would find it helpful to know here what nested breadcrumbs look like (they're visually indicated by the folder icon, right?).

Reply:
As far as I can tell, yes. There's an additional sentence there now: https://github.com/adobe/spectrum-css/pull/2793/commits/26a6353de48dbfe7fd8ec5f99705609761e613a4